src/frontend/JasmineGraphFrontEnd.cpp

```diff
@@ -350,6 +352,47 @@ void *frontendservicesesion(void *dummyPt) {
                 frontend_logger.log("Graph does not exist or cannot be deleted with the current hosts setting",
                                     "error");
             }
+
+        } else if (line.compare(REFORMAT) == 0) {
+            write(connFd, SEND.c_str(), FRONTEND_COMMAND_LENGTH);
+            write(connFd, "\r\n", 2);
+
+            // We get the name and the path to graph as a pair separated by |.
+            char graph_data[300];
+            bzero(graph_data, 301);
+            string name = "";
```

Comment:
Lets declare name and path at line 380. There is no need of declaring them at this line.

Reply:
For example, 

string name = strArr[0];

Reply:
Ok